demo.py

```diff
@@ -0,0 +1,88 @@
+import json
+import requests
+import random
+
+if __name__ == '__main__':
+
+    eai_base = "https://lit-sea-20426.herokuapp.com"
+    dummy_base = "https://stil-taiga-23440.herokuapp.com"
```

Comment:
Are you sure this is the correct URL? I think `stil` should be `still`.

Reply:
Good catch, my bad